src/components/VHeader/VSearchBar/VSearchButton.vue

```diff
@@ -1,15 +1,33 @@
 <template>
-  <VIconButton
-    v-bind="$attrs"
-    :size="`search-${size}`"
-    class="search-button hover:text-white group-hover:text-white hover:bg-pink group-hover:bg-pink p-0.5px ps-1.5px focus:p-0 border md:border border-dark-charcoal-20 hover:border-pink group-hover:border-pink rounded-e-sm active:shadow-ring"
-    :icon-props="{ iconPath: searchIcon }"
-    :aria-label="$t('search.search')"
-    v-on="$listeners"
-  />
+  <div>
+    <VButton
+      v-if="isHomeRoute && isMinScreenMd"
+      v-bind="$attrs"
+      :aria-label="$t('search.search')"
+      class="inline-block cta-search-button font-semibold h-full text-2xl"
+      v-on="$listeners"
+      >{{ $t('search.search') }}</VButton
+    >
+    <VIconButton
```

Comment:
It _kind of_ seems like these should be combinable in some way... Like `VIconButton` should just be an implementation of `VButton` or something?

Probably for a follow up issue 👍